src/ValidatorKeysTool.cpp

```diff
@@ -170,6 +249,40 @@ void signData (std::string const& data,
     std::cout << std::endl;
 }
 
+void generateManifest (
+    std::string const& type,
+    boost::filesystem::path const& keyFile)
+{
+    using namespace ripple;
+
+    auto keys = ValidatorKeys::make_ValidatorKeys (keyFile);
+
+    auto const m = keys.manifest();
+
+    if (m.empty())
+    {
+        std::cout << "The last manifest generated is unavailable. You can\n";
+        std::cout << "generate a new one.\n\n";
+        return;
```

Comment:
Suggestion: Provide the command line to generate a new manifest. eg. `generate a new one with the "create_token" command.`